features/FEATURE_BLE/targets/TARGET_CORDIO/stack/ble-host/include/att_api.h

```diff
@@ -1,20 +1,22 @@
-/*************************************************************************************************/
-/*!
- *  \file
+/* Copyright (c) 2018 Arm Limited
+ * SPDX-License-Identifier: Apache-2.0
  *
- *  \brief  Attribute protocol client and server API.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
  *
- *  Copyright (c) 2009-2019 Arm Ltd. All Rights Reserved.
- *  Arm Ltd. confidential and proprietary.
+ * http://www.apache.org/licenses/LICENSE-2.0
  *
- *  IMPORTANT.  Your use of this file is governed by a Software License Agreement
- *  ("Agreement") that must be accepted in order to download or otherwise receive a
- *  copy of this file.  You may not use or copy this file for any purpose other than
- *  as described in the Agreement.  If you do not agree to all of the terms of the
- *  Agreement do not use this file and delete all copies in your possession or control;
- *  if you do not have a copy of the Agreement, you must contact Arm Ltd. prior
- *  to any use, copying or further distribution of this software.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
  */
+
+/*************************************************************************************************/
+/*!
+ *  \brief Attribute protocol client and server API.
 */
```

Comment:
blame regex, will fix

Reply:
no, really, eclipse regex did this, very unhappy with it now, will write a stern letter to the Eclipse Foundation